lint

```diff
@@ -172,6 +174,34 @@ lint_files() {
     exit $lint_result
 }
 
+matches_any() {
+    local filename="$1"
+    local patterns="$2"
+    IFS=$'\n' # Make newlines the only separator.
+    set -f    # Disable globbing so $patterns can be treated as is.
+    for pattern in $patterns; do
```

Comment:
Thanks! I've switched to `while read -r` and an `herestring`, which should be safer and nicer.